packages/lesswrong/client/useCkEditorInspector.ts

```diff
@@ -0,0 +1,17 @@
+import { RefObject } from "react";
+import { useGlobalKeydown } from "@/components/common/withGlobalKeydown";
+import CKEditorInspector from "@ckeditor/ckeditor5-inspector";
+import type CKEditor from "@/lib/vendor/ckeditor5-react/ckeditor";
+
+export function useCkEditorInspector(editorRef: RefObject<CKEditor<any>>) {
+  useGlobalKeydown(ev => {
```

Comment:
Do we want this accessible to non-admins?  (On windows this also collides with `Save all open tabs as bookmarks in a new folder`: https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Cwebpage-shortcuts, which is not the end of the world since we don't use it but :shrug:)